tf2onnx/onnx_opset/signal.py

```diff
@@ -211,14 +211,14 @@ def any_version(cls, opset, ctx, node, **kwargs):
             "Add", inputs=[real_part2.output[0], imag_part2.output[0]],
             name=utils.make_name('ComplexAbs_' + node.name))
 
-        if opset == 1:
+        if opset < 13:
```

Comment:
Looks like these cases can also be consolidated using graphbuilder since axes is const.

Reply:
I kept this one to reuse ind0 but I can remove it.